README.md

```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that these flags are set for you.
+
+Currently in `Build, Execution, Deployment -> Build Tools -> Gradle -> Runner` (but just search for `runner`)
```

Comment:
Maybe specify this is under "Preferences" (macOS) or "Settings" (Windows/Linux)?

Reply:
Awesome idea - done something similar to what you suggested.